sdk/src/claim.rs

```diff
@@ -862,7 +868,7 @@ impl Claim {
     /// asset_bytes - reference to bytes of the asset
     pub async fn verify_claim_async<'a>(
         claim: &Claim,
-        asset_bytes: &'a [u8],
+        asset_data: &mut ClaimAssetData<'_>,
```

Comment:
What forces the mut here? Is is that the stream needs to be mut?

Reply:
The stream pointers